README.md

```diff
@@ -432,6 +433,79 @@ Example 3:
 
 _________________________
 
+
+### qLoRA Tuning Example
```

Comment:
same nit:
```suggestion
### QLoRA Tuning Example
````

Reply:
I actually think we want to be more specific here. If these docs are just for auto_gptq and not bnb_qlora then we should note that this is 4bit GPTQ-LoRA with AutoGPTQ. Whereas bnb_qlora is 4bit QLoRA with bitsandbytes as both are named QLoRA